include/arch/x86/multiboot.h

```diff
@@ -51,15 +56,21 @@ extern void z_x86_multiboot_init(struct x86_multiboot_info *);
  * is when we want the loader to configure the framebuffer for us.
  */
 
+#define X86_MULTIBOOT_HEADER_FLAG_MEM	(1 << 1)  /* want mem_/mmap_* info */
+#define X86_MULTIBOOT_HEADER_FLAG_FB	(1 << 2)  /* want fb_* info */
```

Comment:
nitpitck, use macro `BIT`. That macro uses proper unsigned type.

Reply:
Fixed, thanks!